Домашнее задание 6. Эмулятор банкомата

LovchikovaES opened the pull request.

LovchikovaES committed:
Домашнее задание 6. Эмулятор банкомата

LovchikovaES committed:
Домашнее задание 6. Эмулятор банкомата
Добавлена проверка на ввод купюр с отрицательным и нулевым количеством, добавлены 2 теста на эти проверки.

kataus commented on hw06-ATM/src/main/java/ru/catn/CellManagerImpl.java:
```diff
@@ -0,0 +1,90 @@
+        createCells();
+    }
+
+    private void createCells() {
```
Мы стараемся следовать следующему порядку методов: public, default, protected, private. Это логично, когда читаешь код - в первую очередь важно что опубликовано, а внутрянки поглубже если кому действительно надо

LovchikovaES replied:
Да, это логично, буду внимательнее к этому. Спасибо ;)

LovchikovaES committed:
Домашнее задание 6. Эмулятор банкомата
Удален класс ATM и изменен порядок методов согласно рекомендациям преподавателя.
